src/org/gk/gkCurator/CuratorActionCollection.java

```diff
@@ -2169,10 +2169,41 @@ private void deleteInstance() {
 		}
 	}
 	
+	/**
+	 * Retrieve a "Create Instance" action for new entities.
+	 * 
+	 * @return Action with specific "SHORT_DESCRIPTION" and "enabled" status
+	 */
 	public Action getCreateInstanceAction() {
-		if (createInstanceAction == null) {
+		// list of constants that curators are not allowed to create new instances of.
+		ArrayList<String> nonCreatable = new ArrayList<String>();
+		nonCreatable.add(ReactomeJavaConstants.Species);
+		nonCreatable.add(ReactomeJavaConstants.Compartment);
```

Comment:
If `nonCreatable` will be a fixed size, an alternative initialization could be:

```
List<String> nonCreatable = Arrays.asList(
    ReactomeJavaConstants.Species, 
    ReactomeJavaConstants.Compartment
);
```

Reply:
```java
List<String> nonCreatable = Arrays.asList(
    ReactomeJavaConstants.Species, 
    ReactomeJavaConstants.Compartment
);
```
This is much better approach!